src/components/common/RSWordleErrorBoundary/index.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+import Button from 'components/common/Button';
+import PageWrapper from 'components/common/PageWrapper';
+import LogoutButton from 'components/LogoutButton';
+import SideNav from 'components/SideNav';
+
+import ErrorBoundaryContext from './context';
+import './styles.css';
+
+class RSWordleErrorBoundary extends React.Component {
+  state = { hasError: false, errorMessage: '' };
+
+  static getDerivedStateFromError(error) {
+    return { hasError: true, errorMessage: error.message };
+  }
+
+  componentDidCatch(error, errorInfo) {
+    console.log({ error, errorInfo });
```

Comment:
```suggestion
    console.error({ error, errorInfo });
```
Maybe it's better this way. The same below